pset-iterators.js

```diff
@@ -160,12 +201,18 @@ Expected Result Shape: A string representing the full name of the president that
 console.log('Problem 6:')
 
 // Add your code below this line
+const firstRepublican = presidents.find(function(president) {
```

Comment:
@Ashbk32 nice work, this works as expected, another way to write the logic in your callback would be 
```
return president.party === 'Republican
```

Both `.find()` and `.filter()` look to evaluate to a true or false, so we can write it like the above without the need of an if statement (even though the way you wrote it, is perfectly fine)

Take a look at the solution for more details: https://github.com/jsd20191008/hw-02-iterators-solution/blob/master/pset-iterators.js#L179